client/src/modules/Registry/components/Form/DoneSection/DoneSection.tsx

```diff
@@ -11,49 +14,52 @@ const arrowSvg = (
   </svg>
 );
 
+const ArrowIcon = (props: Partial<CustomIconComponentProps>) => {
+  return <Icon component={svg} {...props} />;
+};
+
 export function DoneSection() {
   return (
-    <Card
-      style={{
-        width: '100%',
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        justifyContent: 'center',
-        gap: '28px',
-      }}
-    >
-      <Row justify="center">
-        <Image
-          style={{ padding: '4px 0', marginBottom: '28px' }}
-          preview={false}
-          src="https://cdn.rs.school/sloths/stickers/slothzy/image.png"
-          alt="slothzy"
-          width={406}
-          height={366}
-          placeholder={<Skeleton.Image style={{ width: 406, height: 366 }} />}
-        />
-      </Row>
-      <Row style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center', gap: '16px' }}>
-        <Row justify="center">
-          <Title level={3} style={{ margin: 0 }}>
-            Success
-          </Title>
-        </Row>
-        <Row justify="center">
-          <Title level={3} style={{ textTransform: 'uppercase', color: '#1890FF', margin: 0 }}>
-            but
-          </Title>
-        </Row>
-        <Row justify="center">{arrowSvg}</Row>
-        <Row justify="center">
-          <Paragraph style={{ color: 'rgba(0, 0, 0, 0.43)', maxWidth: '480px', textAlign: 'center', margin: 0 }}>
-            {SUCCESS_TEXT}
-          </Paragraph>
-        </Row>
-        <Row justify="center">
-          <Paragraph style={{ color: 'rgba(0, 0, 0, 0.43)', textAlign: 'center', margin: 0 }}>See you soon</Paragraph>
-        </Row>
+    <Card>
+      <Row justify="center" gutter={[0, 28]}>
+        <Col>
+          <Image preview={false} src="https://cdn.rs.school/sloths/stickers/slothzy/image.png" alt="slothzy" />
+        </Col>
+        <Col span={24}>
+          <Row justify="center" gutter={[0, 16]}>
+            <Col span={24}>
+              <Row justify="center">
+                <Title level={3} style={{ marginBottom: 0 }}>
+                  Success
+                </Title>
+              </Row>
+            </Col>
+            <Col span={24}>
+              <Row justify="center">
+                <Title level={3} style={{ textTransform: 'uppercase', color: '#1890FF', marginBottom: 0 }}>
+                  but
+                </Title>
+              </Row>
+            </Col>
```

Comment:
Try this refactoring:
- Col could be used without Row for every line. Grid in antdesign is very flexible
- When Col is used without any `span`, etc., it gets space of child elements

It's just recommendation. If it will not work, you can leave as it is now

```suggestion
          <Row align="center" justify="center" gutter={[0, 16]}>
            <Col>
                <Title level={3}>
                  Success
                </Title>
                <Title level={3} style={{ textTransform: 'uppercase', color: '#1890FF', marginBottom: 0 }}>
                  but
                </Title>
            </Col>
```

Reply:
noted, thanks, gonna leave as it is, ain't working.